### ISSUE TITLE
Database creation+population (via liquibase) missing or not documented for docker deployment

### ISSUE THREAD

**Issue by sheaphillips** [eclipse/kapua#363]
Apologies if I've missed this somewhere else (although I have looked through issues + docs) but when deploying the kapua stack using local docker containers (and presumably in OpenShift) there is no longer a reference/step to populate the database.

As a result when connecting to the broker and attempting to authenticate, I get an error indicating the USR_USER table is not found, etc.

```
Caused by: org.h2.jdbc.JdbcSQLException: Table "USR_USER" not found; SQL statement:
SELECT attributes, created_on, display_name, email, external_id, modified_on, name, optlock, phone_number, properties, status, user_type, created_by, id, modified_by, scope_id FROM usr_user WHERE (name = ?) [42102-193]
```

What is the incantation I need to run liquibase and/or populate the databased through some other means?

**Comment by ctron** [comment]
In general it is not required anymore. Database structures are generated by the broker image at the moment.

I would suggest you build docker images locally or use the one from my repository (ctron). I am working on providing more recent docker images automatically.

**Comment by sheaphillips** [comment]
@ctron I am using fresh locally built docker images, but when I connect to the broker (using paho python client and websocket protocol), and configure the client to auth using default creds ("kapua-sys", "kapua-password") I get the error below (and a bunch more) in the logs of the broker container:

```
20:13:56,163 [qtp1353756631-36] INFO  KapuaSecurityBrokerFilter:307  - User name kapua-sys - client id paho/6D927F52DDF503A441
[EL Info]: 2017-03-08 20:13:56.365--ServerSession(823335029)--EclipseLink, version: Eclipse Persistence Services - 2.6.3.v20160428-59c81c5
[EL Info]: connection: 2017-03-08 20:13:56.707--ServerSession(823335029)--/file:/maven/lib/extra/kapua-user-internal-0.2.0-SNAPSHOT.jar_kapua-user login successful
[EL Warning]: 2017-03-08 20:13:57.04--UnitOfWork(1600372516)--Exception [EclipseLink-4002] (Eclipse Persistence Services - 2.6.3.v20160428-59c81c5): org.eclipse.persistence.exceptions.DatabaseException
Internal Exception: org.h2.jdbc.JdbcSQLException: Table "USR_USER" not found; SQL statement:
SELECT attributes, created_on, display_name, email, external_id, modified_on, name, optlock, phone_number, properties, status, user_type, created_by, id, modified_by, scope_id FROM usr_user WHERE (name = ?) [42102-193]
        at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
        at org.h2.message.DbException.get(DbException.java:179)
        at org.h2.message.DbException.get(DbException.java:155)
        at org.h2.command.Parser.readTableOrView(Parser.java:5389)
        at org.h2.command.Parser.readTableFilter(Parser.java:1257)
        at org.h2.command.Parser.parseSelectSimpleFromPart(Parser.java:1897)
        at org.h2.command.Parser.parseSelectSimple(Parser.java:2045)
        at org.h2.command.Parser.parseSelectSub(Parser.java:1891)
        at org.h2.command.Parser.parseSelectUnion(Parser.java:1709)
        at org.h2.command.Parser.parseSelect(Parser.java:1697)
        at org.h2.command.Parser.parsePrepared(Parser.java:445)
        at org.h2.command.Parser.parse(Parser.java:317)
        at org.h2.command.Parser.parse(Parser.java:289)
        at org.h2.command.Parser.prepareCommand(Parser.java:254)
        at org.h2.engine.Session.prepareLocal(Session.java:561)
        at org.h2.server.TcpServerThread.process(TcpServerThread.java:263)
        at org.h2.server.TcpServerThread.run(TcpServerThread.java:158)
        at java.lang.Thread.run(Thread.java:745)
```

**Comment by sheaphillips** [comment]
@ctron just also tried with latest kapua-broker image from your dockerhub repo and got a broker startup failure:

```
20:18:39,630 [main] ERROR BrokerService:626  - Failed to start Apache ActiveMQ ([kapua, null], java.lang.RuntimeException: org.h2.jdbc.JdbcSQLException: Schema "KAPUADB" not found; SQL statement:
SET SCHEMA kapuadb [90079-193]
        at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
        at org.h2.message.DbException.get(DbException.java:179)
        at org.h2.message.DbException.get(DbException.java:155)
        at org.h2.engine.Database.getSchema(Database.java:1747)
        at org.h2.command.dml.Set.update(Set.java:408)
        at org.h2.command.CommandContainer.update(CommandContainer.java:98)
        at org.h2.command.Command.executeUpdate(Command.java:258)
        at org.h2.engine.Engine.openSession(Engine.java:206)
        at org.h2.engine.Engine.createSessionAndValidate(Engine.java:154)
        at org.h2.engine.Engine.createSession(Engine.java:137)
        at org.h2.server.TcpServerThread.run(TcpServerThread.java:148)
        at java.lang.Thread.run(Thread.java:745)
)
20:18:39,634 [main] INFO  BrokerService:807  - Apache ActiveMQ 5.14.0 (kapua, null) is shutting down
```

Maybe I will try the whole stack from your repo instead of mix of locally-built images with "your" broker image...

**Comment by ccustine** [comment]
I have the same issue as above: "Schema "KAPUADB" not found" with new images from current develop branch. I'm not sure what the issue is but the new behavior from https://github.com/eclipse/kapua/commit/914a83bf5e1179b0616aa09c12c3603c899190ed seems to not work. I also tried setting commons.db.schema but that causes other issues.

I reverted https://github.com/eclipse/kapua/commit/914a83bf5e1179b0616aa09c12c3603c899190ed locally to get by for now.
